README.adoc

```diff
@@ -45,13 +45,12 @@ https://github.com/projectriff/riff-cli[riff-cli] repositories.
 +
 [source, bash]
 ----
-cd $(go env GOPATH)   #defaults to ~/go
-git clone -o upstream https://github.com/projectriff/function-sidecar src/github.com/projectriff/function-sidecar/
-git clone -o upstream https://github.com/projectriff/function-controller src/github.com/projectriff/function-controller/
-git clone -o upstream https://github.com/projectriff/topic-controller src/github.com/projectriff/topic-controller/
-git clone -o upstream https://github.com/projectriff/http-gateway src/github.com/projectriff/http-gateway/
-git clone -o upstream https://github.com/projectriff/shell-function-invoker src/github.com/projectriff/shell-function-invoker/
-git clone -o upstream https://github.com/projectriff/riff-cli src/github.com/projectriff/riff-cli/
+go get github.com/projectriff/function-sidecar
```

Comment:
Running those prints 
```
package github.com/projectriff/function-sidecar: no Go files in /tmp/foofo/src/github.com/projectriff/function-sidecar
```

Need to add `/...` at the end to suppress above message.

Should also add `-d` to not try to install (this creates a binary named `cmd` anyway, which is overwritten by others)

Reply:
Sorry, didn't know you were reviewing - maybe you can apply your changes directly